src/format.ts

```diff
@@ -47,6 +47,5 @@ interface Format {
 
 // ----- Exports ----- //
 
-export { Pillar, Special, Theme, Design, Display };
-
-export type { Format };
+export type { Theme, Format };
```

Comment:
`theme` is a `type`, is this ok?

Reply:
I think so yeah, to access the values that conform to `Theme` you'd look them up on the `Pillar` and `Special` `enum`s.